tools/build_all.py

```diff
@@ -50,10 +50,18 @@ def build_size(example, board):
 def skip_example(example, board):
     ex_dir = 'examples/device/' + example
     board_mk = 'hw/bsp/{}/board.mk'.format(board)
-    for skip_file in glob.iglob(ex_dir + '/.skip.MCU_*'):
-        mcu_cflag = '-DCFG_TUSB_MCU=OPT_' + os.path.basename(skip_file).split('.')[2]
-        with open(board_mk) as mk:
-            if mcu_cflag in mk.read():
+
+    with open(board_mk) as mk:
+        mk_contents = mk.read()
+
+        # Skip ESP32-S2 board if example is not FreeRTOS one
+        if 'freertos' not in example and 'CROSS_COMPILE = xtensa-esp32-elf-' in mk_contents:
```

Comment:
Cross compiler should be xtensa-esp32s2-elf for ESP32-S2.

Reply:
yeah, it is, this is only a ci script to skip non-freertos example in the repo :)

Reply:
ops, I just realize my mistake, sorry, change to esp32s2 now